server/src/stateManagement.mjs

```diff
@@ -274,7 +275,7 @@ function handleDynamicResponseValues(userId, methodName, params, ws, resp){
       };
       const sFcnBody = resp.response + ';' + 'return f(ctx, params);'
       const fcn = new Function('ctx', 'params', sFcnBody);
-      const result = fcn(ctx, params);
+      const result = await fcn(ctx, params);
```

Comment:
We call dynamic functions (YAML) in multiple places. They should all be updated to support async/await

Reply:
Added await/asyn in the other places,where dynamic function is being called